tests/python/unittest/test_operator.py

```diff
@@ -1911,10 +1911,17 @@ def test_bdiv(a, b):
         check_binary_op_forward(c, lambda a, b: a / b, gen_broadcast_data, mx_nd_func=mx.nd.divide)
         check_binary_op_backward(c, lambda g_out, a, b: (g_out / b, - g_out * a / (b * b)), gen_broadcast_data)
 
-    def test_bmod(a, b):
+    def test_bmod(a_, b_):
```

Comment:
Are you using a and a_ on purpose? It seems like sometimes you are using a and b, while other times you use a_ and b_

Reply:
I think a_ and b_ are placeholders here before the sym.cast operator is applied to them and they become a and b.  This enforces that they'll be float64s before the broadcast_mod op is applied, which would otherwise cause numerical issues.  Then we compare the result of the broadcast_mod (c) with the python and numpy versions (which are both passed as lambdas to the check).